notebooks/en/issues_in_text_dataset.ipynb

```diff
@@ -0,0 +1,3635 @@
+{
```

Comment:
<p>Same here. While for the rest of the outputs I encourage you to remove the duplication, you can leave this for pandas dataframes. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>